packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/gux-dropdown-multi.tsx

```diff
@@ -229,7 +230,7 @@ export class GuxDropdownMulti {
   onClearselected(event: CustomEvent): void {
     event.stopPropagation();
 
-    this.updateValue('');
+    this.updateValue(undefined);
```

Comment:
Change was needed based on the change described in the comment on line 358, otherwise the number selected tag was still showing "1" when nothing was selected.
This was the only place value was being set to empty string instead of undefined when empty and it was causing inconsistencies in code and behavior